pkg/minikube/node/start.go

```diff
@@ -176,6 +177,15 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		if err = bs.JoinCluster(*starter.Cfg, *starter.Node, joinCmd); err != nil {
 			return nil, errors.Wrap(err, "joining cluster")
 		}
+
+		cnm, err := cni.New(*starter.Cfg)
+		if err != nil {
+			return nil, errors.Wrap(err, "cni")
+		}
+
+		if err := cnm.Apply(cpr, []cni.Runner{cpr, starter.Runner}); err != nil {
```

Comment:
just a comment,  could starter.Runner be used as cpr too ?

Reply:
`starter.Runner` refers to the newly created node. cpr is the control-plane, which we need to pass in because only the control-plane has kubectl.